gr-uhd/examples/grc/rfnoc_images/x410_X4_200_image_core.grc

```diff
@@ -0,0 +1,533 @@
+options:
```

Comment:
GRC:
![x410_X4_200_image_core](https://github.com/user-attachments/assets/301c7224-b2fb-4ad8-8c29-407438111f5c)

Reply:
Excellent, more of a functional question: "Virtual Port"s in GRC work with NoC domain connections, as well, right? So in theory, we could have a `ce_clk` virtual port connecting to the BSP block's output and to all noc shell wrapper block's inputs?

Reply:
That's a good catch, and something that I didn't think of. It won't work, though, because the `rfnoc_image_builder` would require an update (the GRC file is not parsed by GNU Radio, but by UHD for the image building). I think that's a good change to make in UHD.